app/models/institution.rb

```diff
@@ -292,14 +297,14 @@ def self.postgres_regex_escape(search_term)
       clause <<  'institution_search LIKE UPPER(:institution_search_term)'
     end
 
-    clause << 'UPPER(city) = :upper_search_term'
+    clause << "UPPER(#{city_physical}) = :upper_search_term"
     clause << 'UPPER(ialias) LIKE :upper_contains_term'
-    clause << 'zip = :search_term'
-    clause << 'country LIKE :upper_contains_term'
+    clause << "#{zipcode_physical} = :search_term"
+    clause << "#{country_physical} LIKE :upper_contains_term"
```

Comment:
Can we keep the single quotes?

Reply:
No, the string interpolation wont work with single quotes.